.drone/drone.jsonnet

```diff
@@ -35,12 +35,23 @@ local secret(name, vault_path, vault_key) = {
 local docker_username_secret = secret('docker_username', 'infra/data/ci/docker_hub', 'username');
 local docker_password_secret = secret('docker_password', 'infra/data/ci/docker_hub', 'password');
 
+// secrets for pushing serverless code packages
+local fn_upload_ops_tools_secret = secret('ops_tools_fn_upload', 'infra/data/ci/tempo-ops-tools-function-upload', 'credentials.json');  
+
 // secret needed to access us.gcr.io in deploy_to_dev()
 local docker_config_json_secret = secret('dockerconfigjson', 'secret/data/common/gcr', '.dockerconfigjson');
 
 // secret needed for dep-tools
 local gh_token_secret = secret('gh_token', 'infra/data/ci/github/grafanabot', 'pat');
 
+# gcs buckets to copy serverless functions to
+local gcp_serverless_deployments = [ 
+  { 
+    bucket: 'ops-tools-tempo-function-source',
+    secret: fn_upload_ops_tools_secret.name,
+  },
+];
```

Comment:
Does this array need to be updated every time we add a new cluster/region?

Reply:
This will need to be updated everytime we add a new bucket. I'd like as many clusters as make sense to share the same buckets. Going to play with that some today.